### PR TITLE
Added docs on gr.Examples

### PR DESCRIPTION
**Pull request by abidlabs**
Added a couple of lines to our changelog. Also realized we never documented `gr.Examples`, so I've added a section in our Blocks guides.

Closes: https://github.com/gradio-app/gradio/issues/6389
Closes: https://github.com/gradio-app/gradio/issues/6355

### CODE REVIEW

File: guides/03_building-with-blocks/01_blocks-and-event-listeners.md [hunk 1]
```diff
@@ -129,6 +129,26 @@ $demo_blocks_essay_simple
...
+You can also set `cache_examples=True` similar to `gr.Interface`, in which case two additional arguments must be provided:
+
+* `outputs`: the component or list of components corresponding to the output of the examples
+* `fn`: the function to run to generate the outputs corresponding to the examples
+
+Here's an example showing how to use `gr.Examples` in a `gr.Blocks` app:
+
+$code_calculator_blocks
+
+**Note**: In Gradio 4.0 or later, when you click on examples, not only does the value of the input component update to the example value, but the component's configuration also reverts to the properties with which you constructed the component. This ensures that the examples are compatible with the component even if its configuration hhas been changed.
```
Comment (freddyaboulton) [inline guides/03_building-with-blocks/01_blocks-and-event-listeners.md:148]: Doulble "h" in "hhas"

File: guides/03_building-with-blocks/01_blocks-and-event-listeners.md [hunk 2]
```diff
@@ -129,6 +129,26 @@ $demo_blocks_essay_simple
 
 See how we can configure the Textbox itself through a new `gr.Textbox()` method. The `value=` argument can still be used to update the value along with Component configuration. Any arguments we do not set will use their previous values.
 
+## Examples
+
+Just like with `gr.Interface`, you can also add examples for your functions when you are working with `gr.Blocks`. In this case, instantiate a `gr.Examples` similar to how would instantiate a component. The constructor of `gr.Examples` takes two required arguments:
```
Comment (freddyaboulton) [inline guides/03_building-with-blocks/01_blocks-and-event-listeners.md:134]:
```suggestion
Just like with `gr.Interface`, you can also add examples for your functions when you are working with `gr.Blocks`. In this case, instantiate a `gr.Examples` similar to how you would instantiate any other component. The constructor of `gr.Examples` takes two required arguments:
```

**Review by freddyaboulton — APPROVED** [review]
Couple of nits but looks great @abidlabs !

**Comment by abidlabs** [comment]
Thanks @freddyaboulton!

**Pull request merged**